openbas-model/src/main/java/io/openbas/database/repository/UserRepository.java

```diff
@@ -66,4 +65,17 @@ void createAdmin(
       + "       left join tags tg on usr_tg.tag_id = tg.tag_id"
       + "      group by us.user_id;", nativeQuery = true)
   List<RawUser> rawAll();
+
+  @Query(value = "select us.user_id, us.user_email, " +
+          "us.user_firstname, us.user_lastname, " +
+          "us.user_country, us.user_organization," +
+          "array_remove(array_agg(tg.tag_id), NULL) as user_tags " +
+          "from USERS us " +
```

Comment:
Nitpick : We should have a team discussion on whether or not we do uppercase or lowercase queries. That being said, it would be nicer if your query would choose between uppercase everywhere or lowercase everywhere 🙂

Reply:
We can go further, only keyword should be in uppercase, it could be easier to read. But for now I used lowercase.
Thanks.

Reply:
That's the way I do it but tbh I don't have a strong opinion on this and I'd be fine with any solution as long as we have a common one.